client/packages/programs/src/JsonForms/components/Search/useSearchQueries.tsx

```diff
@@ -91,17 +96,9 @@ export const useSearchQueries = ({
           </Typography>
         );
       },
-      saveFields: saveFields ?? [
-        'id',
-        'code',
-        'firstName',
-        'lastName',
-        'dateOfBirth',
-        'gender',
-        'email',
-        'document',
-      ],
-      placeholderText: placeholderText ?? 'Search by patient code...',
+      saveFields: saveFields ?? null,
```

Comment:
Seemed like "save the whole object" would be a better default if `saveFields` not specifically defined.